pkg/admission/clusterworkspacetype/admission.go

```diff
@@ -83,5 +84,25 @@ func (o *clusterWorkspaceType) Validate(ctx context.Context, a admission.Attribu
 		return errors.New("organization type can only be created in root workspace")
 	}
 
+	if a.GetOperation() != admission.Update {
+		return nil
+	}
+
+	if a.GetOldObject().GetObjectKind().GroupVersionKind() != tenancyv1alpha1.Kind("ClusterWorkspaceTypes").WithVersion("v1alpha1") {
+		return nil
+	}
+	oldU, ok := a.GetOldObject().(*unstructured.Unstructured)
+	if !ok {
+		return fmt.Errorf("unexpected type %T", a.GetOldObject())
+	}
+	old := &tenancyv1alpha1.ClusterWorkspaceType{}
+	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(oldU.Object, old); err != nil {
+		return fmt.Errorf("failed to convert unstructured to ClusterWorkspaceType: %w", err)
+	}
+
+	if !equality.Semantic.DeepEqual(cwt.Spec.Extend, old.Spec.Extend) {
+		return admission.NewForbidden(a, errors.New("spec.extend is immutable"))
```

Comment:
ähm, why?! This cannot be immutable.

Reply:
For now, it is.

Reply:
Can you create a follow-up issue/task in the epic?